staff/edgar-frias-jimenez/precourse/calculator/index.js

```diff
@@ -0,0 +1,30 @@
+// Project 1 - Calculator
+const calculator = (...newArgs) => {
+  let results = [];
+  let isInvalid = false;
+
+  for (let i = 0; i < newArgs.length; i++) {
+    if (typeof newArgs[i] !== 'number' || newArgs[i] === null || newArgs[i] === undefined) {
+      isInvalid = true;
+    }
+  }
+
+  if (newArgs.length === 1 && isInvalid) {
+    console.log('Sorry you entered an invalid input');
+  } else if (newArgs.length === 1 && !isInvalid) {
+    const oneParamResult = Math.sqrt(newArgs[0]);
+    console.log(`As you only entered one input I can only give the square root of ${newArgs[0]}, that is: ${oneParamResult % 1 !== 0 ? oneParamResult.toFixed(3) : oneParamResult}`);
```

Comment:
@edfrias repeat code in line 16, 23, 24, 25 and 26 - better in a function?

Ej: 
```javascript
function functionName(num) {
    return num % 1 !== 0 ? num.toFixed(3) : num;
}

const summatory = functionName(newArgs.reduce((a, b) => a + b));
```

Reply:
@maz-e You are right! If you see in my next commit you'll see that I've created a new function that tries to see if the number has float or not and return the desired value, thanks mate!